wp-sw-cache/wp-sw-cache-admin.php

```diff
@@ -167,9 +167,10 @@ function options() {
       </td>
     </tr>
     <tr>
-      <th scope="row"><label for="wp_sw_cache_name"><?php _e('Current Cache Name', 'wpswcache'); ?></label></th>
+      <th scope="row"><label for="wp_sw_cache_name"><?php _e('Cache Version & Name', 'wpswcache'); ?></label></th>
```

Comment:
From my point of view, allowing name customization is adding unnecessary complexity to the worker. Notice that if you expose this to be configurable, then you need to delete former data after an update or you will leak unreachable resources. But, to be efficient, you probably don't want to re-fetch all the contents but move them to the previous cache to the new. Furthermore, I don't see the advantage of allowing changing the name to a namespace-prefixed database.